Moonlight macOS/Input/Control.m

```diff
@@ -0,0 +1,188 @@
+//
+//  Control.m
+//  Moonlight macOS
+//
+//  Created by Felix Kratz on 15.03.18.
+//  Copyright © 2018 Felix Kratz. All rights reserved.
+//
+
+
+#include "Gamepad.h"
+#include "Control.h"
+#include "Controller.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <math.h>
+#import "ControllerSupport.h"
+#include "Limelight.h"
+
+#ifdef _MSC_VER
+#define snprintf _snprintf
+#endif
+
+static bool verbose = true;
+Controller* _controller;
+ControllerSupport* _controllerSupport;
+
+void onButtonDown(struct Gamepad_device * device, unsigned int buttonID, double timestamp, void * context) {
+    if (verbose) {
```

Comment:
We shouldn't depend on `verbose` being true to set button flags. There are similar bugs in `onButtonUp` and `onAxisMoved`

Reply:
True. Fixed